md/rest_api.md

```diff
@@ -1,4 +1,4 @@
-# REST Api
+# REST API
```

Comment:
this page has been deleted, replaced by open API version, since swagger is no more documented (may require a rebase)